randobot/handler.py

```diff
@@ -282,28 +819,55 @@ async def roll_and_send(self, args, message, encrypt, dev):
                 'Don\'t get greedy!'
             )
             return
-
+        if draft.get('enabled'):
+            if draft.get('race_type') == 'qualifier' or (draft.get('race_type') == 'practice' and draft.get('auto_draft')):
+                if draft.get('race_type') == 'qualifier':
+                    if not can_monitor(message):
+                        await self.send_message(
+                            'Only Race Monitors can roll qualifier seeds.'
+                        )
+                        return
+                    draft.update({'rolled_at': datetime.datetime.now()})
+                await self.handle_random_seed(encrypt, dev, reply_to)
+                return
+            else:
+                if draft.get('status') != 'complete':
+                    await self.send_message(
+                        f'Sorry {reply_to}, drafting must be completed before rolling the seed.'
+                    )
+                    return
+            await self.roll(
+                preset=None,
+                encrypt=encrypt,
+                dev=True, # awaiting 8.0 release
+                reply_to=reply_to,
+                settings=self.patch_settings(dev=True)
+            )
+            if draft.get('race_type') in ('practice', 'tournament'):
+                await self.ex_settings('', '')
```

Comment:
I think you should do this before `self.roll`, because the `roll` method has a sleep period while it waits for the seed to be generated so it can post the hash. Better to post the settings first as seed generation can take a hot minute when the site is busy.